sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -9,10 +9,9 @@
 - Key Phrase Extraction
 - Named Entity Recognition
 - Recognition of Personally Identifiable Information
-- Linked Entity Recognition
-- Extractive Summarization
```

Comment:
It is still accessible through the most recent released client if customers want to try it. I didn't bother to add it here because, as you know, it is not part of the GA API that we will target soon. Do you think there is any drawback to this plan?

Reply:
IMO I think it makes sense to keep it in the beta for the ease of the users who have been using the feature. I know we'll remove it in the GA version, but then plan to add it back in the subsequent beta. Anyone who is using the feature, will need to pin their JS client library to the previous version before the design breaking changes. I think the idea is to migrate users to the new design asap and this might hold people back

Reply:
I am reaching out offline to discuss this further and to clarify the service team plan for ExtSum.

Reply:
Oops just saw this... posted on the channel. Guess we'll get an answer either way.

Reply:
I added it back!